View/time_slider.py

```diff
@@ -30,33 +31,13 @@ def __init__(self, parent):
                                      button_hover_color="dark gray",
                                      progress_color="light gray"
                                      )
-        self._slider.configure(command=self._update_values)
-        self._slider.set(0)
-        self._slider.grid(row=4, column=1, columnspan=2)        
-    
-    def _update_values(self, a):
-        """_summary_
-        Updates hours and minutes based on slider value, as well as notifying observers
-        """
-        self.updating = True
-        self._hour = self.get_hours()
-        self._minute = self.get_minutes()
-        self.formatted_time = self.format_time(self._hour, self._minute, 0)
-        self.notify_observers()
-        self.updating = False
-        
-    def format_time(self, hour, minute, second) -> str:
-        """Formats hours, minutes and seconds as HH:MM:SS.
-
-        Args:
-            hour (int): 
-            minute (int): 
-            second (int): 
-
-        Returns:
-            str: time formatted as "HH:MM:SS"
-        """
-        return str(hour).zfill(2) + ":" + str(minute).zfill(2) + ":" + str(second).zfill(2)
+        self._slider.configure(command=self._update_time_in_model)
+        self._start_value = self.model.get_time().hour*60 + self.model.get_time().minute
+        self._slider.set(self._start_value)
+        self._slider.grid(row=4, column=1, columnspan=2)
+            
+    def _update_time_in_model(self, a):
```

Comment:
What is `a` used for?

Reply:
And is the method needed?

Reply:
I don't really know why but, customTkinter's command attribute in the slider acts weird when I don't use it:
![image](https://github.com/user-attachments/assets/b2a49fb2-36de-4e1a-92a1-10926032d794)
It just acts as a placeholder variable that helps to avoid this error when the slider is updating:
TypeError: TimeSlider.set_time() takes 1 positional argument but 2 were given

The method can definitively be removed